.gitignore

```diff
@@ -1 +1,2 @@
 .vagrant*
+*.elc
```

Comment:
This pattern is **glob** based on [fnmatch(3)](http://man7.org/linux/man-pages/man3/fnmatch.3.html), not regex.

https://git-scm.com/docs/gitignore
> Otherwise, Git treats the pattern as a shell glob: "*" matches anything except "/", "?" matches any one character except "/" and "[]" matches one character in a selected range. See fnmatch(3) and the FNM_PATHNAME flag for a more detailed description.

In this article, examples such as `*.html` and `/*.c` are listed.

Reply:
thanks, I didn't know you could have patterns with + :smile: